numpy/lib/tests/test_io.py

```diff
@@ -1391,6 +1391,21 @@ def test_names_and_comments_none(self):
         control = np.array([(1, 2), (3, 4)], dtype=[('col1', int), ('col2', int)])
         assert_equal(test, control)
 
+    def test_error_ext(self):
+        # gh-13200
+        with tempdir() as tmpdir:
+            with open(os.path.join(tmpdir, "test.csv"), "wb") as f:
+                f.write(b'\xcf\x96')
+
+            with warnings.catch_warnings(record=True) as w:
+                warnings.simplefilter("always")
```

Comment:
Yes and no. They are raised by default, but only in development mode, since we do not want new downstream DeprecationWarnings or so cause failures on user test runs.

Reply:
Good to know. At any rate, `ResourceWarning`s are emitted in a destructor, so would not propagate as errors.